examples/universal-lwmqtt/universal-mqtt.h

```diff
@@ -91,12 +91,12 @@ void connect() {
   mqtt->mqttConnect();
 }
 
-void publishTelemetry(String data) {
-  mqtt->publishTelemetry(data);
+bool punlishTelemetry(String data) {
```

Comment:
Copy / paste typo here and elsewhere in this file, should be `publishTelemetry`.

Reply:
Sorry about the typos :sweat: